client/src/containers/SingleArticle/SingleArticle.js

```diff
@@ -1,27 +1,37 @@
 import React, { Component } from 'react';
-import apiClient from '../../helpers/apiClient';
+import apiClient from '../../helpers/apiClient'
```

Comment:
why did you change this file?? - it should have been working (it call getSingleArticle and return a single article not an array)

Please put it back as it was.

Reply:
these two are the same, so I just left one of them

Reply:
it is alright I thing I have got it